stacks-signer/src/v0/signer.rs

```diff
@@ -194,12 +201,13 @@ impl SignerTrait<SignerMessage> for Signer {
                     self.signer_db
                         .insert_burn_block(burn_header_hash, *burn_height, received_time)
                 {
-                    warn!(
+                    error!(
```

Comment:
What you have is fine by me, but a slightly more Rust-y way of doing this would be:

```rust
self.signer_db
    .insert_burn_blocko(burn_header_hash, *burn_height, receive_time)
    .unwrap_or_else(|e| {
       error!(
           "Failed to write burn block event to signerdb";
           "err" => ?e,
           "burn_header_hash" => %burn_header_hash,
           "burn_height" => burn_height
       );
       panic!("{self} Failed to write burn block event to signerdb: {e}");
    });
```